Stardaze/OutputFormatter.swift

```diff
@@ -381,7 +381,17 @@ internal struct OutputFormatter: Visitor {
     }
 
     internal func visit(_ string: String) -> String {
-        return "\"\(string)\""
+        let forwardSlashesHandled = string.replacingOccurrences(of: "\\", with: "\\\\")
```

Comment:
This is not super pretty, but I can't think of a simpler way to do it. You will always need to map one set of characters with another set of characters.

Reply:
As in, you can't do it with a single regex in anyway that I know